docs/api/asyncio_con.rst

```diff
@@ -550,3 +571,154 @@ Connection Pools
     .. py:attribute:: free_size
 
         Number of available connections in the pool.
+
+
+.. _edgedb-python-asyncio-api-transaction:
+
+Transactions
+============
+
+The most robust way to make transaction is using a retry loop:
```

Comment:
```suggestion
The most robust way to execute transactional code is to use 
the ``retry()`` API:
```

Reply:
I think it's important to still use the word loop, so "the ``retry()`` loop API" would probably be a more informative description.  It is impossible to use "retry()" without a loop, isn't it?

Reply:
Yes. Added "loop"